inspirehep/dojson/common/base.py

```diff
@@ -345,14 +345,16 @@ def urls(self, key, value):
     urls = []
     value = utils.force_list(value)
     for val in value:
-        url_dict = {}
-        description = utils.force_list(val.get('y'))
+        description = val.get('y')
+        if isinstance(description, (list, tuple)):
+            description = description[0]
+            # FIXME: log.warning("Multiple descriptions")
```

Comment:
Mh, I have been thinking about this, and I think this is not the right time to add logging to DoJSON.

We currently have _something_, like https://github.com/inspirehep/inspire-next/blob/df44a93c172d7276fce7882681f0885340ea5be8/inspirehep/dojson/hep/fields/bd1xx.py#L76-L82, but it's not built in the right way. I think that a future PR should add logging where it's needed and fix it where it's broken.

So, for now, please remove this and similar comments.